### ISSUE TITLE
Saving the models' result when tested on the training set and test set

### ISSUE THREAD

**Issue by Shadz13** [massquantity/LibRecommender#27]

**Comment by Shadz13** [comment]
I came across this following line of code in line 246:
```
            if verbose > 1:
                train_loss_str = "train_loss: " + str(
                    round(float(np.mean(train_total_loss)), 4)
                )
                print(f"\t {colorize(train_loss_str, 'green')}")
                # for evaluation
                self._set_last_interacted()
                self.print_metrics(eval_data=eval_data, metrics=metrics)
                print("=" * 30)
```

I revised self.print_metrics(train_data=train_data,eval_data=eval_data, metrics=metrics)
However, the evalresults_train.par is empty.
